### PR TITLE
Setup grunt flow task and run on travis

### PR DESCRIPTION
**Pull request by zpao**
Depends on #6682, currently looking at the root for .flowconfig but can be changed as needed. cc @sebmarkbage

### CODE REVIEW

File: grunt/tasks/flow.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+var grunt = require('grunt');
+
+var extension = process.platform === 'win32' ? '.cmd': '';
+
+module.exports = function() {
+  var done = this.async();
+  grunt.util.spawn({
+    cmd: 'node_modules/.bin/flow' + extension,
+    args: ['check', '.'],
```
Comment (spicyj) [inline grunt/tasks/flow.js]: Will people run this interactively and get confused that it's slow?

**Comment by spicyj** [comment]
Seems good. Grunt is weird.

**Comment by sebmarkbage** [comment]
Ok. If you rebase, it should pass.

File: grunt/tasks/flow.js (hunk 1 above)
Comment (zpao) [inline grunt/tasks/flow.js]: All of our grunt tasks are slow so probably doesn't matter.

**Comment by facebook-github-bot-8** [comment]
@zpao updated the pull request.

**Comment by zpao** [comment]
Alright, locked in at Flow 0.24.0 since that's what we have installed (and our flowconfig expects that). I also ensured we're using latest fbjs which comes with a `Promise.js` lib file with the non-standard methods that we're (where we just means other code in fbjs) using.

**Comment by facebook-github-bot-0** [comment]
@zpao updated the pull request.

**Pull request merged**